modules/ztp-image-based-upgrade-troubleshooting.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-troubleshooting_{context}"]
+= Troubleshooting
+
+You can encounter issues during the image-based upgrade.
+
+[id="ztp-image-based-upgrade-troubleshooting-manual-cleanup_{context}"]
+== `AbortFailed` or `FinalizeFailed` error
+
+During the finalize stage or when you stop the process at the `Prep` stage, normally, {lcao} cleans up the following resources:
+
+* Stateroot that is no longer required
+* Precaching resources
+* OADP CRs
+* `ImageBasedUpgrade` CR
+
+If the {lcao} fails to perform the above steps, it transitions to the `AbortFailed` or `FinalizeFailed` states.
+The condition message and log shows which steps failed.
+
+.Example error message
```

Comment:
Didn't notice it in this PR but just to confirm do we have `must-gather` [documented](https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/must-gather.md)?

Reply:
I haven't added it yet, but it is on my list and will add it in the next couple of days.

Reply:
@pixelsoccupied Added [here](https://github.com/openshift/openshift-docs/pull/74025/commits/f7752668e07f9df6fddc1025d3404f28065153d9) / [preview](https://74025--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/ztp-image-based-upgrade#ztp-image-based-upgrade-troubleshooting-must-gather_ztp-image-based-upgrade), PTAL